xla/service/gpu/fusions/fusions.cc

```diff
@@ -141,6 +141,9 @@ absl::StatusOr<std::unique_ptr<FusionInterface>> GetFusionEmitter(
                          .GetModule()
                          ->config()
                          .debug_options();
+#ifdef TENSORFLOW_USE_ROCM
```

Comment:
Why do we need an ifdef for this?

Reply:
mlir_converter::IsHloConversionSupported always returns false for ROCm. So for ROCm side we are yet to support mlir emitters.  So I added this until we support mlir emitters as tracked in https://github.com/ROCm/frameworks-internal/issues/8063

Reply:
Why not just do it properly right away? See my other comment re `xla_gpu_enable_mlir_emitter_fallback`.

Reply:
Does `xla_gpu_check_mlir_emitter_support` add better clarification?